components/Ink/src/MDCSimpleInkView.m

```diff
@@ -0,0 +1,93 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCSimpleInkView.h"
+#import "MDCSimpleInkGestureRecognizer.h"
+#import "private/MDCSimpleInkLayer.h"
+
+@interface MDCSimpleInkView ()
```

Comment:
Does a simpleinkview have to be in the front of the view it's on? I know I have code in my text fields that's always moving around the subviews in front and behind each other.

Reply:
Not necessarily. You should be able to put the inkview behind other views, however, it may require you to handle touches in your view controller and call startInkAtPoint: and endInkAnimated: as appropriate in your touch handler.